editoast/src/core/stdcm.rs

```diff
@@ -125,6 +127,10 @@ pub enum STDCMResponse {
         departure_time: DateTime<Utc>,
     },
     PathNotFound,
+    Conflicts {
+        pathfinding_result: PathfindingResult,
+        conflicts: Vec<Conflict>,
+    },
```

Comment:
I don't know if this will be done in the following refacto PR. Currently the struct used for the endpoint response is the same as the one used for core response. We should have two different struct now since editoast enrich the response with conflicts.

Reply:
This might take a bit more time to do in this PR. Is it possible to handle it in the refactor PR https://github.com/OpenRailAssociation/osrd/pull/9434 instead? @flomonster